languages/python/BitwardenClient/bitwarden_client.py

```diff
@@ -18,6 +18,12 @@ def password_login(self, email: str, password: str) -> ResponseForPasswordLoginR
         )
         return ResponseForPasswordLoginResponse.from_dict(result)
```

Comment:
Can you remove this method while at it. We don't support password login.

Reply:
Removed those three functions, and the imports that were no longer being used in [4212204](https://github.com/bitwarden/sdk/pull/229/commits/42122044c1c2731c561bc95a62d2a7f2cb67543b)